lib/pliny/tasks/db.rake

```diff
@@ -34,7 +34,7 @@ begin
     task :seed do
       if File.exist?('./db/seeds.rb')
         database_urls.each do |database_url|
-          Sequel.connect(database_url)
+          DB = Sequel.connect(database_url)
```

Comment:
ooh true, this causes a warning to be displayed.

It is intended though, I wanted `db/seeds.rb` to have an instance of `DB` just like the rest of the application. Thoughts?